saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -21,16 +28,57 @@ class BreakerBoard:
     webhook execution.
     """
 
-    def __init__(self, storage: Storage, failure_threshold: int):
-        self.storage = storage
+    def configure(
+        self,
+        storage: Storage,
+        mode: str,
+        failure_threshold: float,
+        cooldown_seconds: int,
+        ttl: int,
+    ):
+        self.failure_threshold = failure_threshold or settings.BREAKER_FAILURE_THRESHOLD
+        self.cooldown_seconds = cooldown_seconds or settings.BREAKER_COOLDOWN
+        self.ttl = ttl or settings.BREAKER_TTL
+        if mode == BreakerMode.FIXED and self.failure_threshold < 1:
```

Comment:
If configuration options are not valid - raise an error, do not let misconfigured Saleor start.

I think we'd rather see Saleor fail right away than find out later that breaker board was misconfigured.